pkg/kubectl/kubectl.go

```diff
@@ -38,6 +38,11 @@ serviceaccounts (aka 'sa'), ingresses (aka 'ing'), horizontalpodautoscalers (aka
 componentstatuses (aka 'cs), endpoints (aka 'ep'), petsets (alpha feature, may be unstable) and secrets.`
 )
 
+// userResources is a group of resources mostly used by a kubectl user
+var userResources = []string{"deployments", "horizontalpodautoscalers",
```

Comment:
make this `[]unversioned.GroupResource`.  `AliasesForResource` can still return a string.